vrs/utils/PixelFrameJpeg.cpp

```diff
@@ -75,7 +75,7 @@ bool PixelFrame::readJpegFrame(const vector<uint8_t>& jpegBuf, bool decodePixels
 
 bool PixelFrame::readJpegFrameFromFile(std::string path, bool decodePixels) {
   FILE* infile = fopen(path.c_str(), "rb");
-  if (infile == NULL) {
+  if (!infile) {
```

Comment:
As a matter of convention, we're explicitly discouraging this type of testing throughout the code base. So you should not find any pointer test like this one, or int implicitly cast to bools either. What could be improved here is testing against `nullptr` rather than `NULL`.